app/controllers/listings_controller.rb

```diff
@@ -484,6 +496,20 @@ def verification_required
 
   private
 
+  def create_bookable(listing_uuid, author_uuid)
+    res = HarmonyService::API::Api.post(:create_bookable,
+                                        body: {
+                                          marketplaceId: @current_community.uuid,
+                                          refId: listing_uuid,
+                                          authorId: author_uuid
+                                        })
```

Comment:
The indentation doesn't look very nice to read. How about an enter before :create_bookable?